packages/shared/src/lib/boot.ts

```diff
@@ -47,8 +47,15 @@ export type BootCacheData = Pick<
 > & { lastModifier?: string };
 
 export async function getBootData(app: string, url?: string): Promise<Boot> {
+  const version = await import('../../../extension/package.json').then(
```

Comment:
Because the boot is something that you need as soon as you start the app, I wouldn't use dynamic import just a regular import here. Also I would consider if this is something we can inject in the build time through webpack magic or something like that and use it as an env var